Updated the second VP message in the issue#147.

Aini-Bashir opened the pull request:
I have updated the description of the VP message and also change the name behind it.

Aini-Bashir committed:
Updated the second VP message in the issue#147.

I have updated the description of the VP message and also change the name behind it.

Remy-Reveltek reviewed (changes requested):
Change this to be a "Personal Statement" or an "Executive Statement" from this person.

Aini-Bashir commented:
@Remy-Reveltek, what do you mean by a "personal statement"?

Remy-Reveltek commented:
@Aini-Bashir, Personal statements are written in the first person, and the opportunity to present an individual and their background.

Paste your text in ChatGpt and ask it to convert it to a "Personal Statement" or an "Executive Statement".

Aini-Bashir committed:
Updated the changes under the issue#147.

I have updated the content according to the "Executive statement" as requested.
